### PR TITLE
k8s 1.30 dependencies updated

### PR DESCRIPTION
**Pull request by vardhaman22**
https://github.com/rancher/rancher/issues/45089
- updated kube dependencies, rancher dependencies and wrangler to v3 
- uses custom forked rancher pkg/apis that can be updated after 1.30 PR gets merged in rancher
- added generation for core and apps packages as well.
Note: factory and interface fille generation is still broken but it helps in updating file imports.
- uses custom fork for aks,eks,gke operators as well but those can be removed once new tags are available for those

### CODE REVIEW

**Comment by chiukapoor** [comment]
> uses custom fork for aks,eks,gke operators as well but those can be removed once new tags are available for those

The latest operator tags with k8s 1.30 related changes are updated in custom forked rancher pkg/api. Use of custom fork for aks, eks, gke is removed.

File: extensions/clusters/gke/gke_cluster_config.go
Comment (igomez06) [inline extensions/clusters/gke/gke_cluster_config.go:241]: Was this changed in pkg/apis?

**Comment by floatingman** [comment]
@chiukapoor Were any provisioning tests run with these changes? Specifically RKE1 and RKE2 tests. It's essential to verify that these changes don't break existing tests.

**Review by igomez06 — COMMENTED** [review]
can we get some sanity checks on provisioning as well? Specifically rke1 and maybe just k3s. Thanks!

**Comment by chiukapoor** [comment]
> @chiukapoor Were any provisioning tests run with these changes? Specifically RKE1 and RKE2 tests. It's essential to verify that these changes don't break existing tests. If so, will you provide links in Slack to tests that were run in CI?

@igomez06 @floatingman The [CI](https://drone-pr.rancher.io/rancher/rancher/40239) has successfully passed for PR https://github.com/rancher/rancher/pull/45483 which is using base branch of this [PR](https://github.com/vardhaman22/shepherd/tree/v1.30.0-rancher5) for shepherd import.

File: extensions/kubeconfig/kubeconfig.go
Comment (chiukapoor) [reply extensions/kubeconfig/kubeconfig.go:29]: The return type has recently been changed by `client-go`.

Reference PR: https://github.com/kubernetes/kubernetes/pull/119398

**Comment by floatingman** [comment]
> > @chiukapoor Were any provisioning tests run with these changes? Specifically RKE1 and RKE2 tests. It's essential to verify that these changes don't break existing tests. If so, will you provide links in Slack to tests that were run in CI?
> 
> @igomez06 @floatingman The [CI](https://drone-pr.rancher.io/rancher/rancher/40239) has successfully passed for PR [rancher/rancher#45483](https://github.com/rancher/rancher/pull/45483) which is using base branch of this [PR](https://github.com/vardhaman22/shepherd/tree/v1.30.0-rancher5) for shepherd import.

I don't think drone runs any provisioning tests.   We specifically want to verify tests in 
https://github.com/rancher/rancher/tree/release/v2.9/tests/v2/validation/provisioning isn't broken by these changes. The RKE1 and K3S test suites from this test package are good sanity checks to run.

File: extensions/kubeconfig/kubeconfig.go
Comment (igomez06) [reply extensions/kubeconfig/kubeconfig.go:29]: sounds good

**Review by igomez06 — APPROVED** [review]
LGTM just waiting on @floatingman 's approval

File: pkg/schemas/management.cattle.io/v3/schema.go
Comment (floatingman) [inline pkg/schemas/management.cattle.io/v3/schema.go:314]: Does this need to be commented out or can it be deleted?

**Review by floatingman — CHANGES_REQUESTED** [review]
There is some commented out code I'd like you to address.

File: pkg/schemas/management.cattle.io/v3/schema.go
Comment (vardhaman22) [reply pkg/schemas/management.cattle.io/v3/schema.go:314]: this is related to psp as well..in this [PR](https://github.com/rancher/shepherd/pull/179/files#diff-b9538a1c8a175ff43d3187a23d5d6b6c488f87d53e9e38a8acba2235688326b2) there is a discussion about the commented code and since those lines were not deleted till now i also commented the code instead of deleting those lines. 
@caliskanugur could you please check once.
Thanks

**Review by floatingman — APPROVED** [review]
LGTM

**Comment by slickwarren** [comment]
merging this to unblock the team since approval has been given already above

**Pull request merged**
